projects/sandbox/caliptra.md

```diff
@@ -0,0 +1,48 @@
+# Project application: **Caliptra**
+
+## Application
+
+### Sandbox application
+
+* Project name: Caliptra
+* Project repo(s):
+
+| Repo | What |
+| ---- | ---- |
+| [Primary Repo](https://github.com/chipsalliance/caliptra)        | This repo, with admin boilerplate and docs |
+| [Caliptra RTL](https://github.com/chipsalliance/caliptra-rtl)    | The primary repository with immutable RTL |
+| [Caliptra SW](https://github.com/chipsalliance/caliptra-sw)      | Caliptra software (ROM, FMC, runtime firmware), and libraries/tools needed to build and test |
+| [Caliptra U-Reg](https://github.com/chipsalliance/caliptra-ureg) | General-purpose libraries and tools for manipulating MMIO registers from Rust |
+| [Caliptra DPE](https://github.com/chipsalliance/caliptra-dpe)    | An implementation of a TCG DICE Protection Environment profile |
+* Brief summary of the project: The Caliptra project focuses on development of HW and SW IP for [The Caliptra Root of Trust](https://github.com/chipsalliance/Caliptra/blob/main/doc/Caliptra.md).
+* Project's open source license: **ASL 2.0**
+* Link to issue tracker: **TBD**
```

Comment:
Should we tweak the main README.md and/or this .md with link to each issue tracker per repo?

Reply:
I would say the main README, seems cleaner. Can I just put a link to the README?